src/build.d

```diff
@@ -876,6 +875,24 @@ auto getHostCXX()
         static assert(false, "Unrecognized or unsupported OS.");
 }
 
+/*
+Gets a string describing the type of host C++ compiler
+
+Returns: a string describing the type of host C++ compiler
+*/
+auto getHostCXXKind()
+{
+    version(Posix)
+    {
+        auto cxxVersion = execute([getHostCXX, "--version"]).output;
+        return !cxxVersion.find("gcc", "Free Software")[0].empty ? "g++" : "clang++";
+    }
+    else version(Windows)
+        return "dmc";
```

Comment:
Couldn't this be MVSC too?

Reply:
I say no, because `msvc-dmc` is a wrapper around MSVC to make it behave like DMC, so for all intents and purposes we're dealing with DMC.